src/InertiaHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace BoxyBird\Inertia;
+
+class InertiaHelper
```

Comment:
Feels like plural "Helpers" would've been more accurate here?

Reply:
Sure, my thought process regarding helpers is that I have one "helper class", but I can see how it could be name to represent multiple helper functions instead. 

At any rate seeing how only one of those methods seem to be in use since the latest commits I'd say it's @boxybird 's decision to stick with a helper class or rename/replace it with whatever he seems fit.

cheers !